docs/source/routing/self-hosted/index.mdx

```diff
@@ -21,7 +21,7 @@ flowchart LR;
 
 <Note>
 
-Self-hosting the GraphOS Router requires a current GraphOS [Enterprise or Free plan](https://www.apollographql.com/pricing).
+Because GraphOS Router supports a collection of features specific to GraphOS, self-hosting GraphOS Router requires a current GraphOS [Enterprise or Free plan](https://www.apollographql.com/pricing). Learn more about [GraphOS Router features](/graphos/routing/graphos-features).
```

Comment:
Given that the Scale plan is coming out soon, and other plans may change, do we just want to simplify this:

```suggestion
Because GraphOS Router supports a collection of features specific to GraphOS, self-hosting GraphOS Router requires a GraphOS [plan](https://www.apollographql.com/pricing). Learn more about [GraphOS Router features](/graphos/routing/graphos-features).
```